INTEGRATING.md

```diff
@@ -0,0 +1,14 @@
+## Integrating Prometheus
+
+Prometheus integrates with the following
+
+1. Any charm that supports the `prometheus_scrape` interface in
+serving as a scrape target for Prometheus. This integration is done
+using the Prometheus Charm library. The documentation of the charm
+library, available through
+[Charmhub](https://charmhub.io/prometheus-k8s) provides further
+details on how this integration may be done.
+
+2. Grafana
+
+3. Alertmanager
```

Comment:
Through which relations? Follow-up issue?

Reply:
@rbarry82 @dstathis Suggestions on what would you like to see here ? Alternatively should I leave this as is for now and you create a PR that includes this when you push Grafana and Alertmanager libraries into the Prometheus repo ?

Reply:
I can work with @rbarry82 and @dstathis to ensure this is addressed now or picked up in a future PR.